lib/pengine/pe_output.c

```diff
@@ -11,6 +11,25 @@
 #include <crm/common/iso8601_internal.h>
 #include <crm/pengine/internal.h>
 
+#define s_if_plural(i) (((i) == 1)? "" : "s")
```

Comment:
As an aside, I've so far resisted putting this in a header because I think it's hacky, but I haven't come up with anything better. Maybe it should be in crm/common/internal.h. Not as part of this PR, though.

Even more aside, it's very difficult to generalize this problem. One approach is to have a dictionary of singular/plurals, e.g. a hash table like spelling["resource"] = [ "resource", "resources" ]. That can handle English well but not other languages where e.g. the verb must change to match the number, or there can be different endings for 1, 2, and more than 2. The simplest approach would probably be to just use "resource(s)" but I hate how that looks. :)

Reply:
I really didn't want to add that define to yet another location, but oh well.  It seemed easier than finding a place to share it.

The typical way to handle this would be the ngettext function, but we don't use gettext at all.  I think we'd probably have to do a lot of work throughout if we wanted to support translations.  I haven't wanted to wade into pacemaker's translation history.  We dealt with a lot of this kind of stuff on anaconda so if we ever want to work on it here, I could probably help out.  If you're curious, here's a document I started a long time ago talking about anaconda translation issues (some of which would apply here too):  https://fedoraproject.org/wiki/Anaconda/Devel/Translation